tests/unit/LHNFilterTest.js

```diff
@@ -306,15 +305,14 @@ describe('Sidebar', () => {
                         ...LHNTestUtils.getAdvancedFakeReport(...boolArr),
                         policyID: policy.policyID,
                     };
-                    const sidebarLinks = LHNTestUtils.getDefaultRenderedSidebarLinks();
+                    const sidebarLinks = LHNTestUtils.getDefaultRenderedSidebarLinks(report1.reportID.toString());
```

Comment:
Similar thinking here, but why isn't the `reportID` a string in the first place? `getFakeReport()` should return it as a string (maybe you're just copying what was there before and I missed these on another PR).

Reply:
I think you're right that we don't need the cast. But I assumed here that the `toString()` was necessary since it's already in the test here:

https://github.com/Expensify/App/blob/23ab71636b9a9f7523ccf9cb2dbdd90357059051/tests/unit/LHNFilterTest.js#L317